docs/spfx/viva/design/design-intro.md

```diff
@@ -5,7 +5,7 @@ ms.date: 12/28/2021
 ms.prod: sharepoint
 ms.localizationpriority: high
 ---
-# Designing Viva Connections custom cards for your dashboard
+# Design guidance for Viva Connections cards
```

Comment:
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.